app/controllers/application_controller.rb

```diff
@@ -19,6 +19,16 @@ class ApplicationController < ActionController::API
     Breakers::OutageException
   ].freeze
 
+  VERSION_STATUS = {
+            alp: 'Alpha',
+            bet: 'Beta',
+            rel: 'Release Candidate',
+            cur: 'Current Version',
+            pre: 'Previous Version',
+            eol: 'End of Life',
+            dep: 'Depricated'
```

Comment:
Depricated -> Deprecated

Reply:
Well this was the PR that I said we'd get to you guys so we could have that conversation, let me know what you'd prefer to see in this case.

Reply:
Fixed